go/cmd/dolt/commands/diff_output.go

```diff
@@ -195,78 +170,37 @@ func (t tabularDiffWriter) Close(ctx context.Context) error {
 	return nil
 }
 
-func (t tabularDiffWriter) BeginTable(ctx context.Context, td diff.TableDelta) error {
+func (t tabularDiffWriter) BeginTable(fromTableName, toTableName string, isAdd, isDrop bool) error {
 	bold := color.New(color.Bold)
-	if td.IsDrop() {
-		_, _ = bold.Printf("diff --dolt a/%s b/%s\n", td.FromName, td.FromName)
+	if isDrop {
+		_, _ = bold.Printf("diff --dolt a/%s b/%s\n", fromTableName, fromTableName)
 		_, _ = bold.Println("deleted table")
-	} else if td.IsAdd() {
-		_, _ = bold.Printf("diff --dolt a/%s b/%s\n", td.ToName, td.ToName)
+	} else if isAdd {
+		_, _ = bold.Printf("diff --dolt a/%s b/%s\n", toTableName, toTableName)
 		_, _ = bold.Println("added table")
 	} else {
-		_, _ = bold.Printf("diff --dolt a/%s b/%s\n", td.FromName, td.ToName)
-		h1, err := td.FromTable.HashOf()
-
-		if err != nil {
-			panic(err)
-		}
-
-		_, _ = bold.Printf("--- a/%s @ %s\n", td.FromName, h1.String())
-
-		h2, err := td.ToTable.HashOf()
-
-		if err != nil {
-			panic(err)
-		}
-
-		_, _ = bold.Printf("+++ b/%s @ %s\n", td.ToName, h2.String())
+		_, _ = bold.Printf("diff --dolt a/%s b/%s\n", fromTableName, toTableName)
```

Comment:
is `diff --dolt` actually a thing?! I see we print that already, I'm just not sure what the purpose of --dolt is in diff output.

Reply:
I guess it's to "match" similar `git diff` output:

```
$ git diff
diff --git a/go/cmd/dolt/commands/show.go b/go/cmd/dolt/commands/show.go
index 68057a357..360659a42 100644
--- a/go/cmd/dolt/commands/show.go
+++ b/go/cmd/dolt/commands/show.go
@@ -17,6 +17,8 @@ package commands
 import (
        "context"
        "fmt"
+       "github.com/gocraft/dbr/v2"
+       "github.com/gocraft/dbr/v2/dialect"
        "regexp"
        "strings"
        "time"
```